### DISCUSSION TITLE
Why  HmGet().Result()  not return error of pb.Nil?

### DISCUSSION THREAD

**Discussion by Harlan6** [redis/go-redis#1992] [Q&A]
for example:
		result, err := redis.RedisFacade().Client.HMGet(context.Background(), "aaafdfewr", "bb").Result()
		fmt.Println(sliceCmd.Err() == redis.Nil)

Q:
I query a not exist key ,but  "sliceCmd.Err() == redis.Nil"  return false ,I just want to know when hash key not exist,is there any other method ?

**Comment by vmihailenco** [comment]
See https://redis.uptrace.dev/guide/server.html#redis-nil and https://redis.io/commands/HMGET
